main.nf

```diff
@@ -477,6 +489,12 @@ ch_cadd_wg_snvs_tbi = params.cadd_wg_snvs_tbi ? Channel.value(file(params.cadd_w
 ch_pon = params.pon ? Channel.value(file(params.pon)) : "null"
 ch_target_bed = params.target_bed ? Channel.value(file(params.target_bed)) : "null"
 
+// parameters to experiment with Control-FREEC
+cf_window = params.cf_window ? Channel.value(params.cf_window) : "null"
+cf_coeff = params.cf_coeff ? Channel.value(params.cf_coeff) : "null"
+cf_ploidy = params.cf_ploidy ? Channel.value(params.cf_ploidy) : "null"
```

Comment:
```suggestion
```
You don't use that anywhere, you use `params.cf_window`, `params.cf_coeff` and `params.cf_ploidy` in the ControlFREEC process.
So I think we can remove it